Route/PathInfoBuilderSlashes.php

```diff
@@ -26,11 +26,12 @@ public function build(AdminInterface $admin, RouteCollection $collection)
     {
         $collection->add('list');
         $collection->add('create');
-        $collection->add('batch', null, array(), array('_method' => 'POST'));
+        $collection->add('batch', null, array(), array(), array(), '', array(), array('POST'));
```

Comment:
> until we drop support for sonata admin 2.*

I forgot you still supported it, sorry. You should consider dropping it, it must hinder development a lot.